### PR TITLE
session start on restart

### PR DESCRIPTION
**Pull request by melindaloubser1**
run `action_session_start` on `restart`

### CODE REVIEW

File: actions/actions.py [hunk 1]
```diff
@@ -549,3 +551,17 @@ async def run(
...
+        return "action_restart"
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[EventType]:
+
+        return [Restarted(),FollowupAction("action_session_start")]
```
Comment (b-quachtran) [inline actions/actions.py:567]: Code linting throwing two errors: missing a space after the comma and missing a newline after the return call

Comment (melindaloubser1) [inline actions/actions.py:567]:
```suggestion
        return [Restarted(), FollowupAction("action_session_start")]
       
```

File: actions/actions.py [hunk 2]
```diff
@@ -549,3 +551,18 @@ async def run(
...
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[EventType]:
+
+        return [Restarted(), FollowupAction("action_session_start")]
+
```
Comment (b-quachtran) [inline actions/actions.py:568]: Linting error from the extra spaces on the last line.

Comment (melindaloubser1) [reply actions/actions.py:568]: urg I should add a hook to the repo so it blacks things on commit, I do this too often

**Comment by melindaloubser1** [comment]
> This line in the test story needs to be updated because of the added synonyms:
> 
> pay_cc: i want to pay off my [gringots]
> ⬇️
> pay_cc: i want to pay off my [gringots]{"entity": "credit_card", "value": "gringotts"}

Accidentally made the change on master, so I just rebased.

File: actions/actions.py [hunk 3]
```diff
@@ -8,6 +8,8 @@
     EventType,
     ActionExecuted,
     SessionStarted,
+    Restarted,
+    FollowupAction
```
Comment (b-quachtran) [inline actions/actions.py:12]:
```suggestion
    FollowupAction,
```

File: actions/actions.py (hunk 1 above)
Comment (b-quachtran) [inline actions/actions.py:567]: Missing new line after the return

Comment (melindaloubser1) [reply actions/actions.py:567]:
```suggestion
        return [Restarted(), FollowupAction("action_session_start")]
       
```

**Pull request merged**